no-computer.md

```diff
@@ -1,55 +1,68 @@
 # No computer project
-
+Baseball! It's TOTALLY a DSL.
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+This language would be good for people who want to run baseball simulations to ask "what if" questions. If a pitcher throws the ball a certain way to a certain location to a certain hitter who swings a certain way, what will happen?
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This idea would help baseball executives and casual fans alike entertain possibilities of things that would happen in a game. For an employee of a baseball team, it would help them prepare for games and opposing pitchers/hitters by "seeing what would happen" ahead of time. It could also help settle petty arguments between fans or help them see what would happen if their favorite player did something specific.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate because so much language and terminology is specific to baseball. We can define these things specifically in a DSL, which makes it seem like the right approach. We can define the field, the players, the strike zone, and so much more as well.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I LOVE baseball and this is totally something I would use! I think baseball, which is one of the stats-heaviest sports I can think of, lends itself well to computation and simulation well.
```

Comment:
Seriously check out Strat-O-Matic. Something tells me this will be right up your alley.